x-pack/legacy/plugins/reporting/server/usage/get_reporting_usage.ts

```diff
@@ -126,6 +150,17 @@ export async function getReportingUsage(
           aggs: {
             [JOB_TYPES_KEY]: { terms: { field: JOB_TYPES_FIELD, size: DEFAULT_TERMS_SIZE } },
             [STATUS_TYPES_KEY]: { terms: { field: STATUS_TYPES_FIELD, size: DEFAULT_TERMS_SIZE } },
+            [STATUS_BY_APP_KEY]: {
+              terms: { field: 'status', size: DEFAULT_TERMS_SIZE },
+              aggs: {
+                jobTypes: {
+                  terms: { field: JOB_TYPES_FIELD, size: DEFAULT_TERMS_SIZE },
+                  aggs: {
+                    appNames: { terms: { field: OBJECT_TYPES_FIELD, size: DEFAULT_TERMS_SIZE } }, // NOTE Discover/CSV export is missing the 'meta.objectType' field, so Discover/CSV results are missing for this agg
```

Comment:
To make it clear, this aggregation is not supporting metrics for the way we store CSV reports, but all the other types of reports are supported.

See the Jest snapshot for what it looks like for CSV